CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 3.7.2 (2019-12-18)
+
+- Fixed the repeater item component to prevent throwing errors when using the keyboard. [#104](https://github.com/blackbaud/skyux-lists/pull/104)
+- Fixed the repeater item component to properly align context menus when chevrons are not present. [#101](https://github.com/blackbaud/skyux-lists/pull/101)
```

Comment:
I would say "to properly align header items" since this affects the header text, context menu, checkbox, and reorder handle

Reply:
Done.